src/components/Layer/Layer.js

```diff
@@ -1,38 +1,101 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import { connect } from 'react-redux';
-
-import core from 'core';
 
+import Icon from 'components/Icon';
 import Input from 'components/Input';
 
+import core from 'core';
```

Comment:
Can you remove this as it's not used in this file

Reply:
Done